Documentation/ExtensionArchitecture/HowTo/Localization/Php.rst

```diff
@@ -0,0 +1,28 @@
+.. include:: /Includes.rst.txt
+.. index::
+   Localization; PHP
+.. _extension-localization-php:
+
+====================
+Localization in PHP
+====================
+
+Sometimes you have to localize a string in PHP code, for
+example inside of a controller or a ViewHelper.
+
+.. todo: Add information on ViewHelper and non-Extbase context
+
+Localization in Extbase context
+===============================
+
+In Extbase context you can use the method
+`\TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate($key, $extensionName)`.
+
+This method requires the localization key as the first and the extension's
+name as the second parameter. Then the corresponding text in the current
```

Comment:
```suggestion
name as optional second parameter. Then the corresponding text in the current
```

Actually, there are some arguments more, the 3rd parameter can be helpful when building a link to a detail page, at least, this should be mentioned.

Reply:
I prepared an example in examples extension to demonstrate using arguments in LocalizationUtility::translate. Will make a follow up once the example is merged